extensions/amp-vimeo/0.1/amp-vimeo.js

```diff
@@ -306,13 +248,13 @@ class AmpVimeo extends AMP.BaseElement {
       // that would disable autoplay on iOS.
       return;
     }
-    this.sendCommand_(this.setVolumeMethod_(), '0');
+    this.sendCommand_('setVolume', '0');
```

Comment:
I'm not sure why we originally generate `setVolume`, but since it's the only use I thought it would be unnecessarily complicated to keep.